dist/angular-selector.js

```diff
@@ -22,6 +22,7 @@
 				value:                 '=model',
 				disabled:              '=?disable',
 				multiple:              '=?multi',
+				createCustom:          '=?create',
```

Comment:
Simply, use the `create` property, not as an optional bindable attribute, but as an optional function:
```javscript
create: '&?'
```
In this way, users are able to provide a personal customized function in order to complete a perfect match between input text and option object, like this:
```javascript
$scope.myCreateFn = function (input) {
	return {
		value: input,
		label: 'New input: ' + input,
		group: 'custom options'
	};
};
```